rasa/shared/utils/validation.py

```diff
@@ -154,9 +154,15 @@ def validate_yaml_schema(yaml_file_content: Text, schema_path: Text) -> None:
         PACKAGE_NAME, SCHEMA_EXTENSIONS_FILE
     )
 
+    # Load schema content using our YAML loader as `pykwalify` uses a global instance
+    # which can fail when used concurrently
+    schema_content = rasa.shared.utils.io.read_yaml_file(schema_file)
+    schema_utils_content = rasa.shared.utils.io.read_yaml_file(schema_utils_file)
```

Comment:
alternatively we can convert all schema files to `json` as this would achieve the same effect and might be better re performance

Reply:
Or even better: define the schemas as Python objects - this saves us reading things entirely. What do you think?

Reply:
(this is related to my insights in https://github.com/RasaHQ/rasa/issues/7892#issuecomment-778302866)

Reply:
I think it might make sense to define schemas as Python objects (don't see any need to go with JSON in this case) — but we can also create an issue and do it later.

Reply:
True. I'll add a changelog, create a new issue + merge 👍🏻

Reply:
https://github.com/RasaHQ/rasa/issues/7977